LBHackney-IT/lbh-social-care
# fixed show of the postError and updated headers

**What**  
- BUG: When the APIs are failing on add allocation, closing the modal or changing worker/team doesn't hide the error message.
- Updated modal headers

<img width="554" alt="Screenshot 2021-01-28 at 21 50 00" src="https://user-images.githubusercontent.com/435399/106203186-4c312580-61bb-11eb-81fe-3d5f8722089e.png">


**Extras**
Fixed props and noise with tests

**How to test it**
- mock the `POST /allocations` and make it fails
- try to add a worker
- the error message is displayed
- change worker, team, or close the modal
- the error disappears
